translations/account/certificates/ceremonyConfirmationCard/th.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "ขอแสดงความยินดีกับ{{name}} ที่ได้รับประกาศเกียรติคุณในปีนี้!",
+  "desc": {
+    "eventDetails": "กรุณายืนยันสถานะการเข้าร่วมพิธีรับประกาศเกียรติคุณ ปีการศึกษา 2567 หากคุณยืนยันเข้าร่วมกิจกรรม คุณจะต้องเข้าร่วมการซักซ้อมการเข้ารับประกาศเกียรติคุณในวันที่ 3 มีนาคม 2568 โดยหากไม่มาซักซ้อม คุณจะถูกตัดสิทธิ์เข้าร่วมพิธี ในวันที่ 11 มีนาคม 2568 ",
+    "ifDecline": "หากยืนยันไม่เข้าร่วมกิจกรรม คุณไม่จำเป็นต้องเข้าร่วมการซักซ้อม ในวันที่ 3 มีนาคม 2568 และไม่มีสิทธิ์ในการเข้าร่วมพิธีในวันที่ 11 มีนาคม 2568",
+    "ifIneligible": "หากคุณยืนยันไม่เข้าร่วมกิจกรรม คุณสามารถรับเกียรติบัตรได้ที่ห้องสมุดในภายหลัง"
+  },
+  "response": {
+    "pending": {
+      "text": "คุณจะเข้าร่วมพิธีประกาศเกียรติคุณหรือไม่?",
+      "action": {
+        "yes": "เข้าร่วม",
+        "no": "ไม่เข้าร่วม"
+      }
+    },
+    "accept": {
+      "text": "คุณได้ยืนยันที่จะ<0>เข้าร่วมพิธี</0>แล้ว <1>เปลี่ยนใจ?</1>",
```

Comment:
This is weird and I have no idea how to make it not.

_P.S. Have you guys gotten a UX writer yet?_